tedium.js

```diff
@@ -203,13 +290,14 @@ Promise.resolve().then(() => {
   const promises = [];
 
   for (const ghRepo of ghRepos) {
-    promises.push(rateLimit(100).then(() => {
+    promises.push(Promise.resolve().then(() => {
```

Comment:
Chatted it out in person. This pattern (here and elsewhere in tedium) is to coerce any promise-like objects returned by various APIs (like node-promisify) into standard ES6 Promise objects.